src/persist-txn/src/operator.rs

```diff
@@ -566,7 +566,7 @@ impl DataSubscribe {
             let (mut data, mut txns) = (ProbeHandle::new(), ProbeHandle::new());
             let data_stream = data_stream.flat_map(|part| {
                 let part = part.parse();
-                part.map(|((k, v), t, d)| {
```

Comment:
Since we own `part` here, we should be able to grab the inner part and skip the clone.

(Personally I would be tempted to avoid the `Deref` implementations and either add explicit methods or make the fields public. But if you prefer to keep the `Deref`s an `From` impl would also do this job!)

Reply:
Done!